### ISSUE TITLE
Renvra pregnancy bug

### ISSUE THREAD

**Issue by eldestdawn** [Fenoxo/sourceTiTS#450]
At the end of the Double Trouble scene this message popped up.

Something bad happened!

Please report this message, and include any prior scene text or a description of what you did before seeing this message:

Version: 0.6.3

Error: Unexpected pregnancy slot used to call tryKnockUp.
	at classes::Creature/tryKnockUp()

**Comment by Fenoxo** [comment]
Does this always happen when do that scene? If so, can we have your save file? (Also, including the last few lines from the scene would make it wayyyy easier to dig out of the code).

**Comment by Jacques00** [comment]
I think this has something to do with the renvraDoubleTrouble() not passing on the "x" value--making it undefined when renvraSuperCumPartII() asks for it.

Maybe the solution is for renvraDoubleTrouble() to pass the value through on the "Next button": 
```
addButton(0,"Next",renvraSuperCumPartII, x);
```

And have:
```
public function renvraSuperCumPartII():void
```

be:
```
public function renvraSuperCumPartII(x:int = -1):void
```

Hope this helps!

**Comment by eldestdawn** [comment]
It was happening all the time, I can't check right now because I cannot access the tits website. I get this when I try to open the main page
"This webpage is not available

ERR_CONNECTION_TIMED_OUT

Google Chrome could not load the webpage because www.taintedspace.com took too long to respond. The website may be down, or you may be experiencing issues with your Internet connection."

This is the only web site among my bookmarks that is giving me crap so I don't think it's my end.

Once the site is back up I can see if it's happening on the newest build or if it was fixed with all the other bugs that gedan hunted down.

**Comment by eldestdawn** [comment]
The door clicks and your eyes flutter open. Renvra is standing there, smirking at you.

“Awake now, spacer?” she asks.

You realize you’re still bent over her desk. Your gut throbs and moving brings a twinge of dull ache; you try to stand up but you’re practically glued to the furniture by sticky pink slime. Your belly sloshes, and the enormous loads of Renvra’s sexual fluids that were pumped into you come back in an aggravating flash.

“Not that you don’t look hot draped on my desk with my cum drooling from your hole...” the half-myr begins, brushing your skin with her fingers almost affectionately, “... but you should throw some panties on and get the fuck out - people in the neighborhood are starting to wonder why they never saw you leave the store.”

You nod dumbly, still high as a kite on myr venom, and collect your gear. Renvra steps aside as you move to take the door handle, but then grabs you and pulls you into a kiss.

“Come back soon,” she whispers.

Something bad happened!

Please report this message, and include any prior scene text or a description of what you did before seeing this message:

Version: 0.6.6

Error: Unexpected pregnancy slot used to call tryKnockUp.
	at classes::Creature/tryKnockUp()
	at classes.Characters::PlayerCharacter/loadInCunt()
	at classes::TiTS/renvraSuperCumPartII()
	at classes::TiTS/buttonClick()


This happens when my character gets to the end of double trouble scene. My character is a half ausar trap (female looking with no cunt, a cock and balls) and is currently incubating the nyrean queen's spawn...i say incubating because it's hard to call it being pregnant without a womb. I think that's the problem with this scene, it's checking for a womb when there is none. i can only get this to happen without a pussy. when i get the character a pussy with one of the various drugs sera has, i can go through the scene with no problems.

**Comment by Jacques00** [comment]
Yes, it just didn't pass in a variable that was needed (a vagina index number--so it can identify what womb/ass to load the cum into for possible pregnancy). So it will fail all the time. This issue should be fixed with the pull request (#457). 

Hope this helps!

**Comment by eldestdawn** [comment]
Seems like it was fixed, i'll close it now.
